shuffle/cli/Cargo.toml

```diff
@@ -12,6 +12,9 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.38"
 bcs = "0.1.2"
+chrono = "0.4"
+chrono-tz = "0.6.0"
```

Comment:
Let's remove unnecessary dependencies (chrono*) and simplify the timestamp to be UTC.

We can do this with the rust stdlib:

https://doc.rust-lang.org/std/time/struct.SystemTime.html#examples

```rust
use std::time::SystemTime;

match SystemTime::now().duration_since(SystemTime::UNIX_EPOCH) {
    Ok(n) => println!("1970-01-01 00:00:00 UTC was {} seconds ago!", n.as_secs()),
    Err(_) => panic!("SystemTime before UNIX EPOCH!"),
}
```